### ISSUE TITLE
amp-analytics signal is unreliable/not working at all

### ISSUE THREAD

**Issue by kamtschatka** [ampproject/amphtml#4925]
## Short description of your issue:

I tried to create a page with an amp-analytics tag and checked how reliable the signals are.
- When there is no config file is included as attribute, the signal is sent out most of the times. 
- When a config file is included as attribute, the signal is NEVER sent out.

### When no config file is provided:
1. Go to [http://jsbin.com/buyiri/edit?html,output](http://jsbin.com/buyiri/edit?html,output)
2. Edit the json for the amp-analytics element and observe the requests in the developer tools
3. Requests to "pageview" are there most of the times, but sometimes they are missing. Note: This seems to only happen in Chrome(i used 53.0.2785.101 m), it worked in Firefox(45.3.0)

### When a config file is provided:
1. Go to [http://jsbin.com/zuxovar/edit?html,output](http://jsbin.com/zuxovar/edit?html,output)
2. Insert correct credentials for a soasta mpulse account or enter a page that returns a config. I fell back to the soasta mpulse config to make sure it is not caused by my config.

Here is what i was using for my tests:
`{"appId":"appid","timestamp":1473679354229}`

Here is what is being returned by soasta mpulse:
`{
"requests": {
"beacon_url": "<key>.mpstat.us/"
},
"vars": {
"h.key": "<apikey>",
"h.d": "<domain>",
"h.cr": "<key>",
"h.t": "1473679354229"
}
}`

3. Edit the json a few times to trigger reloads. If you have a look at the developer tools, amp is making a request to the config and then stops(This happens on chrome and firefox for me)
4. I have tried to debug a little bit, but i got stuck. I can see that amp is actually preparing the url for the request, but the request is never sent.

## What browsers are affected?

As mentioned before: The issue with the config file occurs on Chrome and Firefox(latest versions) and the issue without the config file occurs only in Chrome and only rarely.

## Which AMP version is affected?

AMP Version: Powered by AMP ⚡ HTML – Version 1472695982886

**Comment by avimehta** [comment]
hi @kamtschatka I am not able to reproduce the error from either of the links that you provided. The page with config file always sends a hit but unfortunately I can't replicate your exact setup because I don't have a valid appid to use. @rudygalfi can you get in touch with mpulse to get a testing account? 

Also, @kamtschatka Do you do anything specific when the hits stop going out? It is possible that there is some prerendering related bug or some page loading race condition that triggers only in a few cases but I have not seen it so far. If you find any concrete steps, please do chime in. I'll keep looking for a little more time.

**Comment by kamtschatka** [comment]
No i did not do anything specific.
The strange thing was: I tried narrowing down the problem a bit more and removed the network timing parameters one by one until it worked again. At that point i did the opposite test to see if that really was the problem and added them again one by one and it kept working until a certain point and suddenly it stopped working again.
I did not figure out what exactly the problem was, but i realized that with the full querystring, it never worked(at least on my machine with 2 different browsers)

I did another test just now with a correct mPulse id and i was able to reproduce this issue on my machine at home. The request to the amp-config.json file is made and then nothing else happens. Note: If you do not have the appID, the request to apm-config.json will fail with error 400 and the signal is sent out without any problems.
